src/app/pages/gdpr/gdpr-page.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { SharedModule } from 'ish-shared/shared.module';
+
+import { GDPRDataRequestComponent } from './gdpr-data-request/gdpr-data-request.component';
+
+const gdprPageRoutes: Routes = [
+  {
+    path: 'confirmRequest',
```

Comment:
if there is only one page under gdpr-requests there is no subpage necessary (e.g. like login), so path should be ''